### DISCUSSION TITLE
It Possible traslate "Kc Error Message" in UI

### DISCUSSION THREAD

**Discussion by jorozcof** [keycloak/keycloak#19945] [Q&A]
The next message is a "kc-error-message". It happens when I try to link an email account that does not exist in the registered users. The flow that I am using is "first broker login"

Is there a possibility to translate the message? In the realm settings I have Default locale in Spanish but the internationalization is not applicable here.

![image](https://user-images.githubusercontent.com/115189644/234322270-ad48e7c3-9bd1-4a13-aee3-0243fa8c90f7.png)

**Comment by jonkoops** [comment]
Looks like [this label](https://github.com/keycloak/keycloak/blob/78958ae4348f879bf91ea59fae6c8a793ccf8bd2/themes/src/main/resources/theme/base/login/messages/messages_en.properties#L263) has not been translated yet. Feel free to open up a pull request to add these missing translations to the [Spanish locale](https://github.com/keycloak/keycloak/blob/main/themes/src/main/resources-community/theme/base/login/messages/messages_es.properties).
